sn_networking/src/lib.rs

```diff
@@ -385,6 +385,13 @@ impl Network {
             let res = self
                 .put_record_once(record.clone(), verify_store.clone())
                 .await;
+
```

Comment:
Can we also add a comment stating that we might only perform this if `verify_store == true`. Or maybe add `if verify_store == true?`